scripts/install-npm-packages-for-integration-tests

```diff
@@ -11,6 +11,9 @@ for dir in packages/*; do
   popd > /dev/null
 done
 
+# Fix issues with failing installation due to polyfills in graceful-fs package
+curl -XGET 'https://raw.githubusercontent.com/isaacs/node-graceful-fs/168bdb8f0bb3174e8499d4bc5878deead4172c39/polyfills.js' > node_modules/npm/node_modules/graceful-fs/polyfills.js
```

Comment:
Can you check if this is still needed? For example using `yarn why graceful-fs` to check which versions are installed.

Reply:
I can confirm this is still an issue.

Steps I took:

- comment off line 15
- Removed the node_modules folder and yarn.lock file
- Changed the resolution of `graceful-fs` to `4.2.4` (and also repeated for `4.2.8` and `4.2.10`) in package.json
- Ran `yarn install` 
- Ran `yarn test:integration`

Result:
The test failed when installing dependencies due to `graceful-fs` [incompatibility](https://app.circleci.com/pipelines/github/commercetools/nodejs/2450/workflows/f1988a69-b803-4977-ad67-800e838bd15a/jobs/19946?invite=true#step-107-869) with node >= 14.x

Suggestion:
Download and keep local copy of the [polyfills.js](https://raw.githubusercontent.com/isaacs/node-graceful-fs/168bdb8f0bb3174e8499d4bc5878deead4172c39/polyfills.js) file within the repo then `cat` and `write` the content it into the failing `npm` polyfills.js module.

<details>

`yarn list --pattern graceful-fs`

warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^5.0.5"
warning Resolution field "dot-prop@6.0.1" is incompatible with requested version "dot-prop@^5.1.0"
warning Resolution field "diff@5.0.0" is incompatible with requested version "diff@^4.0.1"
warning Resolution field "graceful-fs@4.2.4" is incompatible with requested version "graceful-fs@~4.1.11"
warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "lodash@4.17.21" is incompatible with requested version "lodash@^3.10.1"
warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "dot-prop@6.0.1" is incompatible with requested version "dot-prop@^4.2.1"
warning Resolution field "y18n@3.2.2" is incompatible with requested version "y18n@^4.0.0"
warning Resolution field "dot-prop@6.0.1" is incompatible with requested version "dot-prop@^5.3.0"
warning Resolution field "lodash@4.17.21" is incompatible with requested version "lodash@4.17.4"
warning Resolution field "diff@5.0.0" is incompatible with requested version "diff@3.2.0"
warning Resolution field "growl@1.10.5" is incompatible with requested version "growl@1.9.2"
warning Resolution field "serialize-javascript@6.0.0" is incompatible with requested version "serialize-javascript@^2.1.2"
warning Resolution field "diff@5.0.0" is incompatible with requested version "diff@^4.0.2"
├─ @types/graceful-fs@4.1.5
└─ graceful-fs@4.2.4

</details>